tools/cli/lib/api.c

```diff
@@ -251,19 +251,19 @@ TDNFCliInfoCommand(
     {
         pPkg = &pPkgInfo[dwIndex];
 
-        printf("Name          : %s\n", pPkg->pszName);
-        printf("Arch          : %s\n", pPkg->pszArch);
-        printf("Epoch         : %d\n", pPkg->dwEpoch);
-        printf("Version       : %s\n", pPkg->pszVersion);
-        printf("Release       : %s\n", pPkg->pszRelease);
-        printf("Install Size  : %s (%u)\n", pPkg->pszFormattedSize, pPkg->dwInstallSizeBytes);
-        printf("Repo          : %s\n", pPkg->pszRepoName);
-        printf("Summary       : %s\n", pPkg->pszSummary);
-        printf("URL           : %s\n", pPkg->pszURL);
-        printf("License       : %s\n", pPkg->pszLicense);
-        printf("Description   : %s\n", pPkg->pszDescription);
+        pr_stdout("Name          : %s\n", pPkg->pszName);
+        pr_stdout("Arch          : %s\n", pPkg->pszArch);
+        pr_stdout("Epoch         : %d\n", pPkg->dwEpoch);
+        pr_stdout("Version       : %s\n", pPkg->pszVersion);
+        pr_stdout("Release       : %s\n", pPkg->pszRelease);
+        pr_stdout("Install Size  : %s (%u)\n", pPkg->pszFormattedSize, pPkg->dwInstallSizeBytes);
+        pr_stdout("Repo          : %s\n", pPkg->pszRepoName);
+        pr_stdout("Summary       : %s\n", pPkg->pszSummary);
+        pr_stdout("URL           : %s\n", pPkg->pszURL);
+        pr_stdout("License       : %s\n", pPkg->pszLicense);
+        pr_stdout("Description   : %s\n", pPkg->pszDescription);
```

Comment:
Does it make sense to suppress the output here? The point of the whole command is to print this out.

Reply:
The idea here is, if '-q' argument is given, don't print anything except errors. If you all think otherwise, I will just revert the changes in this function.

Reply:
FWIW, dnf does show the help output with `-q`. Similar for the `info` command.

Reply:
Thank you, I have made the necessary changes. Please let me know if something needs to be printed irrespective of '-q' option.